sdk/spring/azure-spring-cloud-autoconfigure/README.md

```diff
@@ -32,6 +32,7 @@ Note that there is no need to add a `version` as those are managed already by th
 ## Key concepts
```

Comment:
Update ci.yml.

Refs: https://github.com/Azure/azure-sdk-for-java/pull/18916/files#diff-b077b3306d6790ea775af4bf780eae1c65a8ea8bce5bb0d7a53d42bec7199c57L170